From 31a0f542e17eb13cc83f5fbb803ac2f40977b550 Mon Sep 17 00:00:00 2001
From: Nickey Yang <nickey.yang@rock-chips.com>
Date: Fri, 10 Jul 2020 10:44:41 +0800
Subject: [PATCH] ARM: dts: rockchip: disable logo for
 rv1126-evb-ddr3-v12-tb-emmc

There is no uboot in the thunder boot flow,
So disable logo for kernel display drivers fully initialized

Signed-off-by: Nickey Yang <nickey.yang@rock-chips.com>
Change-Id: Ib6d36fc606dd8a4170a0a12c0800768d4f2af64a
---
 arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-emmc.dts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-emmc.dts b/arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-emmc.dts
index b415078fb37a..817f5c8b468d 100644
--- a/arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-emmc.dts
+++ b/arch/arm/boot/dts/rv1126-evb-ddr3-v12-tb-emmc.dts
@@ -140,3 +140,10 @@
 	memory-region-thunderboot = <&rkisp_thunderboot>;
 };
 
+&route_dsi {
+	status = "disabled";
+};
+
+&vcc18_lcd_n {
+	/delete-property/ regulator-boot-on;
+};
-- 
2.35.3

